pkgs/development/python-modules/audiotools/default.nix

```diff
@@ -7,26 +7,56 @@
   AudioToolbox,
   AudioUnit,
   CoreServices,
+  pkg-config,
+  libmpg123,
+  lame,
+  twolame,
+  libopus,
+  opusfile,
+  libvorbis,
+  libcdio,
+  libcdio-paranoia,
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "audiotools";
-  version = "3.1.1";
+  version = "3.1.1-unstable-2020-07-29";
   pyproject = true;
 
-  build-system = [ setuptools ];
-
-  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
-    AudioToolbox
-    AudioUnit
-    CoreServices
+  build-system = [
+    setuptools
+    pkg-config
```

Comment:
`build-system` is only meant to contain build-time python dependencies.
Build-time non-python dependencies should be placed into `nativeBuildInputs`.

Reply:
Fixed